bfvmm/src/serial/src/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ include(${CMAKE_INSTALL_PREFIX}/cmake/CMakeGlobal_Includes.txt)
 
 list(APPEND SOURCES
     serial_port_ns16550a.cpp
+    serial_port_pl011.cpp
```

Comment:
This should probably on be added to systems (like ARM) that need this hardware

Reply:
I feel like it's adding unnecessary complexity to detect and branch on this. The PL011 code is written in an architecture-dependent manner - sure, only ARM systems have it, but it's not hurting anything sticking around. It's fairly small code too. If you really want to do this though, we should probably discuss how we'll go about switching on architecture in cmake - should probably include @JaredWright in that discussion too.

Reply:
Point taken. Likely we will need something as we cannot compile all of the ARM logic on Intel and vice versa.

Reply:
+1